src/src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import styled from 'styled-components'
```

Comment:
I actually know the guy that wrote emotion so I tend to use it 😄https://emotion.sh/